xbmc/FileItem.cpp

```diff
@@ -3634,3 +3637,8 @@ bool CFileItem::GetCurrentResumeTimeAndPartNumber(int64_t& startOffset, int& par
   }
   return false;
 }
+
+bool CFileItem::SetStreamDetailsFromVideoPlayer(const VideoStreamInfo& videoInfo, int videoDuration, const AudioStreamInfo& audioInfo, const SubtitleStreamInfo& subtitleInfo)
```

Comment:
drop. CFileItem should not know about VideoPlayer

Reply:
Then I have to move this code (setting stream details) to VP. Is it ok?

fileItem.GetVideoInfoTag()->m_streamDetails.SetStreams(videoInfo, videoDuration, audioInfo, subtitleInfo);